### ISSUE TITLE
Silly test needs to be fixed

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#118]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/118) by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

@jojo978  reports that there's a ridiculous pair of tests in test_core.py

```text
Hi Joe,

What do you mean by
self.assertGreater(ds.saved_auto_cuts.cuts_prm["postpeak_deriv"], 0) 
in a test below? It seems like you're comparing a tuple with an int.

Young
```

```python
def test_make_auto_cuts(self):
    """Make sure that non-trivial auto-cuts are generated and reloadable from file."""
    data = self.load_data()
    data.summarize_data()
    data.auto_cuts(forceNew=True, clearCuts=True)
    ds = data.first_good_dataset
    self.assertLess(ds.cuts.good().sum(), ds.nPulses)

    data2 = self.load_data(clear_hdf5=False)
    for ds in data:
        self.assertGreater(ds.saved_auto_cuts.cuts_prm["postpeak_deriv"], 0)
        self.assertGreater(ds.saved_auto_cuts.cuts_prm["pretrigger_rms"], 0)
```

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Fixes #118, a pointless test that failed on Py 3.5.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Removing milestone: v0.6 (automated comment)
